### ISSUE TITLE
[BUG] Backups made by cron not visible in volume details

### ISSUE THREAD

**Issue by franta** [longhorn/longhorn#4358]
## Describe the bug

Previous backups  created by the scheduler are not visible

## To Reproduce

Not visible:

![image](https://user-images.githubusercontent.com/739017/182849949-0ed9b126-c2aa-455f-84bd-0c13a130985a.png)

but backups are created:

![image](https://user-images.githubusercontent.com/739017/182850254-1cefdd54-d1c9-4e3b-b9db-a412ad4518bd.png)

## Expected behavior

Backups are visible in volume detail, not only the last one.

In the case of manually created backups, they are all visible.

## Environment

 - Longhorn version: 1.2.4
 - Installation method Helm
 - Kubernetes distro RKE v1.22.7-rancher1-2
   - Number of management node in the cluster: 3
   - Number of worker node in the cluster: 3
 - Node config
   - OS type and version: Centos 7
   - CPU per node: different between clusters, all are wrong
   - Memory per node: different between clusters, all are wrong
   - Disk type SSD
   - Network bandwidth between the nodes: 10Gbps
 - Underlying Infrastructure VMWare
 - Number of Longhorn volumes in the cluster: independent, usually between 10-20

Backups via nfs on synology.

**Comment by innobead** [comment]
cc @mantissahz @smallteeths

**Comment by mantissahz** [comment]
Hi @franta, 
Recurring job with `backup` job type will take a new snapshot to back up and clean up the last snapshot then leave the new snapshot. We retain the backups only on remote backup target.

**Comment by franta** [comment]
Hi @mantissahz  ,
thanks for explaining, change makes sense because snapshots consume space on nodes.

Question. Is there some option to keep snapshots available? Users know the previous way, I think version 1.1.something, when there were also snapshots. And it's a bit inconsistent because manually created backups keep snapshots.

Thanks.

**Comment by franta** [comment]
And one another note, the header of the section is Snapshots and Backups. It makes sense in v1.1 but for now it's only Snapshots.

**Comment by mantissahz** [comment]
@franta No, there is no such option to keep the snapshots of previous backups now, maybe it could be a future improvement. Backups will keep the latest one snapshot for the recurring job or a manually creating.
